digits/dataset/generic/views.py

```diff
@@ -10,6 +10,7 @@
 import caffe_pb2
 import flask
 import matplotlib as mpl
+import matplotlib.pyplot as plt
```

Comment:
FYI, I learned recently that you can do this:
```py
>>> import matplotlib as mpl
>>> mpl.pyplot
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'pyplot'
>>> import matplotlib.pyplot
>>> mpl.pyplot
<module 'matplotlib.pyplot' from '/usr/lib/python2.7/dist-packages/matplotlib/pyplot.pyc'>
```
That would let you change less code, but I think it's debatable whether that's more legible. Up to you.